code/drasil-example/Drasil/NoPCM/GenDefs.hs

```diff
@@ -45,7 +45,7 @@ roc_temp_simp_deriv_sentences = map foldlSentCol [
     QT.temp vol [makeRef2S newA3, makeRef2S newA4, makeRef2S newA5],
   genDefDesc5 density mass vol]
 
-genDefDesc1 :: (HasShortName x, Referable x) => x -> UnitalChunk -> [Sentence]
+genDefDesc1 :: (HasUID x, HasShortName x, Referable x) => x -> UnitalChunk -> [Sentence]
```

Comment:
And this one too? Though we might consider making `HasUID` a pre-condition of `Referable`.